main.go

```diff
@@ -31,7 +31,23 @@ func main() {
 	defer db.Close()
 
 	db.AutoMigrate(&Fruit{})
+	
+	// Delete all data in Fruit table
+	db.Delete(&Fruit{})
 
+	// Sample data
+	_, err  = db.CreateFruit(Fruit{ID: 1, Name: "Apple", Stock: 10})
```

Comment:
Can you remove the ID value from all the objects, please?

The ID column is auto increment. If we explicitly set it then we won't be able to insert a new entry in the table.
https://github.com/golang-starters/golang-http-crud/blob/f56f9cbae6d2375b2cf449d0064ba7feaacdeb3d/repository.go#L23-L27

The database throws a 
```
(pq: duplicate key value violates unique constraint "fruits_pkey") 
```
when I try to insert a new key.
```suggestion
	_, err  = db.CreateFruit(Fruit{Name: "Apple", Stock: 10})
```